imap_processing/spice/time.py

```diff
@@ -1 +1,72 @@
 """Time conversion functions that rely on SPICE."""
+
+import typing
+from collections.abc import Collection
+from typing import Union
+
+import numpy as np
+import numpy.typing as npt
+import spiceypy as spice
+
+from imap_processing.spice import IMAP_SC_ID
+from imap_processing.spice.kernels import ensure_spice
+
+TICK_DURATION = 2e-5  # 20 microseconds
```

Comment:
Where did this value come from?

Reply:
That is from the SCLK kernel. I looked for spice functions to retrieve that value and it can be gotten programmatically, but had a small error associated with doing it that way.
```
print(spice.sct2e(-43, 1) - spice.sct2e(-43, 0))
print(20e-6)

2.002716064453125e-05
2e-05
```